project.clj

```diff
@@ -86,6 +91,7 @@
                          [puppetlabs/jdbc-util "0.6.2"]
                          [puppetlabs/typesafe-config "0.1.5"]
                          [puppetlabs/ssl-utils "0.8.3"]
+                         [puppetlabs/jruby-utils "0.6.0"]
```

Comment:
I was thinking jruby-utils would only be brought in as a dependency by OSS puppetserver.  Unless it is being used somewhere else as well, do you think this needs to be added to clj-parent?